Cargo.toml

```diff
@@ -54,104 +54,104 @@ pallet-nfts = { path = "pallets/nfts", default-features = false }
 pallet-myth-proxy = { path = "pallets/myth-proxy", default-features = false }
 
 # External pallets
-pallet-collator-staking = { git = "https://github.com/blockdeep/pallet-collator-staking.git", tag = "v1.2.5", default-features = false }
+pallet-collator-staking = { git = "https://github.com/alexggh/pallet-collator-staking.git", branch = "polkadot-stable2412-2-with-backport-8040", default-features = false }
```

Comment:
Will be changed once https://github.com/blockdeep/pallet-collator-staking/pull/44 gets merged.